### PR TITLE
script that will create a branch on a github repository

### PR DESCRIPTION
**Pull request by macdiesel**
@edx/pipeline-team PTAL.

### CODE REVIEW

File: scripts/cut_branch.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+            rc=target_branch,
+            sha=commit_hash,
+            msg=commit_message
+        )
+    )
+    try:
+        github_api.delete_branch(target_branch)
+    except GithubException:
+        LOG.error("Unable to delete branch {branch_name}. " +
+                  "Will attempt to recreate"
```
Comment (feanil) [inline scripts/cut_branch.py:104]: This message is a bit misleading. You mean it's going to try creating the new branch right?  Is it possible to check for branch existence before creating it?

**Review by feanil — COMMENTED** [review]
One question, otherwise seems good.

File: scripts/cut_branch.py (hunk 1 above)
Comment (macdiesel) [reply scripts/cut_branch.py:104]: We can, but again could we defer this work to a subsequent PR.

Comment (feanil) [reply scripts/cut_branch.py:104]: I think if you take out the 'will attempt to recreate' part I'm good with the rest of this.

**Review by feanil — APPROVED** [review]

**Pull request merged**